release-scripts/do_release.py

```diff
@@ -91,6 +91,18 @@
     (re.compile(r'__version__\s*=\s*".*"'),
      '__version__ = "{version-no-v}-dev"'),
 
+    (re.compile(r'__libnetwork_plugin_version__\s*=\s*".*"'),
```

Comment:
I don't understand what you mean by your last sentence. Do we want to have the versions from the last (previous) release? I thought we wanted to have the versions of the most recent release.

These versions are only used in calicoctl/__init__.py.  This does not update any other version strings in the project.  The values from this file that are used outside of the file are the calicoctl version (in the same-named command) and the kubernetes/rkt versions to set the default plugin version to pull.

Should we be adding `-dev` to the names of the libnetwork, libcalico, and felix version strings to show that it is not bound to a release?